Resources/doc/reference/action_list.rst

```diff
@@ -525,6 +525,8 @@ You can :
 
 - `header_style`: Customize the style of header (width, color, background, align...)
 - `header_class`: Customize the class of the header
+- `cell_style`: Customize the style of inner cells (width, color, background, align...)
```

Comment:
I added this for convenience following the existing option `header_style`. But if it has to be discussed, i think that it's better to do this in a other PR. This is not the purpose of this one, so i removed this.